pkg/base-dev/mar/story.hoon

```diff
@@ -0,0 +1,70 @@
+/-  *story
+/+  *story
+!:
+|_  tale=story
++$  story-diff       [additions=story deletions=story]
+::
+++  grad
+  |%
+  ++  form  %story-diff
```

Comment:
I think we're going to want a simple mark file for %story-diff so that they can be sent over the wire.  It can be very simple, just needs a conversion to/from noun.

The case where it might be needed is when you do a nontrivial merge of a story file and then try to sync that desk over the network.

Reply:
done in https://github.com/urbit/urbit/pull/5612/commits/b43b9f195e12e6313d5eb68e401cfe85c324cc93